src/main/java/jeryl/fyp/logic/parser/ParserUtil.java

```diff
@@ -135,4 +140,98 @@ public static ProjectStatus parseProjectStatus(String projectStatus) throws Pars
+        String trimmedDeadlineName = deadlineName.trim();
+        if (!DeadlineName.isValidDeadlineName(trimmedDeadlineName)) {
+            throw new ParseException(ProjectName.MESSAGE_CONSTRAINTS);
+        }
+        return new DeadlineName(trimmedDeadlineName);
+    }
+
+    /**
+     * Converts a datetime string to a LocalDateTime Object.
+     * @param dateTime A string in format: yyyy-MM-dd hh:mm.
+     * @return A LocalDateTime variable implied by the string.
+     * @throws ParseException if the given {@code dateTime} is invalid.
+     */
+    public static LocalDateTime parseDeadlineDatetime(String dateTime) throws ParseException {
+        String[] dateAndTime = dateTime.trim().split(" ");
+        String hour;
+        String min;
+        if (dateAndTime.length != 2) {
+            throw new ParseException("The correct Format: YYYY-MM-DD HH:mm");
+        } else {
+            String[] components = dateAndTime[1].strip().split(":", 2);
+            if (components.length != 2) {
+                throw new ParseException("The correct Format: YYYY-MM-DD HH:mm");
+            } else {
+                hour = components[0];
+                min = components[1];
+                if (hour.length() == 2 && min.length() == 2) {
+                    try {
+                        LocalDate date = parseLocalDate(dateAndTime[0]);
+                        LocalTime time = LocalTime.parse(dateAndTime[1]);
+                        return LocalDateTime.of(date, time);
+                    } catch (Exception e) {
+                        throw new ParseException("The correct Format: YYYY-MM-DD HH:mm. \nHH should be a number in "
+                                + "[0,23]; mm should be a number in [0,59]");
+                    }
+                }
+                throw new ParseException("The correct Format: YYYY-MM-DD HH:mm");
+            }
+        }
+    }
+
+    /**
+     * Converts a datetime string to a LocalDate Object.
+     * @param date A string in format: yyyy-MM-dd.
+     * @return A LocalDate variable implied by the string.
+     * @throws ParseException
+     */
+    public static LocalDate parseLocalDate(String date) throws ParseException {
+        String[] components = date.strip().split("-", 3);
+        String year;
+        String month;
+        String day;
+        if (components.length != 3) {
+            throw new ParseException("The correct Format: YYYY-MM-DD HH:mm");
+        } else {
+            year = components[0];
+            month = components[1];
+            day = components[2];
+            if (year.length() == 4 && month.length() == 2 && day.length() == 2) {
+                try {
+                    return LocalDate.parse(date);
+                } catch (Exception e) {
+                    throw new ParseException("The correct Format: YYYY-MM-DD HH:mm");
+                }
+            }
+            throw new ParseException("The correct Format: YYYY-MM-DD HH:mm");
+        }
+    }
```

Comment:
Do you think it's better to accept multiple formats? E.g. [iP](https://github.com/RussellDash332/ip/blob/master/src/main/java/stashy/parser/Parser.java#L40)